apps/CMakeLists.txt

```diff
@@ -26,3 +26,11 @@ target_link_libraries(rt_retexture
         VC::meshing
         VC::texturing
 )
+
+add_executable(rt_brisk_match src/BriskMatch.cpp)
+target_link_libraries(rt_brisk_match
+        opencv_core
+        opencv_imgcodecs
+        opencv_features2d
+        opencv_highgui
+        )
```

Comment:
Four-space indent for the libs. No space indent for the bracket. (Ignore that the ones above yours are also wrong. I think those are fixed in a different branch.)